content-learn/mastering-meshery-istio/introduction-to-service-meshes/linkerd/getting-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+doctype: "Chapter"
+chapterTitle: "Getting Started"
+description: "Meshery is the service mesh managemen plane which offers lifecycle, configuration, and performance management of service meshes and their workloads."
+videos: 6
+lectures: 8
+---
+
+import { ChapterStyle } from "../../../../src/components/Learn-Components/Chapters-Style/chapters.style.js";
+import LinkerdAdapter from "../../../../src/assets/images/learn-layer5/linkerd/linkerd-adapter.png";
+import InstallLinkerd from "../../../../src/assets/images/learn-layer5/linkerd/install-linkerd.png";
+
+<ChapterStyle>
+
+<h2 class="chapter-sub-heading">Deploy Linkerd</h2>
+
+With <a href="https://github.com/layer5io/linkerd-service-mesh-workshop/blob/master/prereq/README.md">prerequisites</a> complete, you are ready to download and deploy Linkerd resources.
```

Comment:
There shouldn't be any reference to the Github repo.

Reply:
You can change the sentence to what we have in istio, that will work too.

Reply:
ok